composer.json

```diff
@@ -14,21 +14,21 @@
     "source": "https://github.com/AOEpeople/asdis"
   },
   "require": {
-    "php": "^7.4 | ^8.0 | ^8.2",
-    "typo3/cms-core": "^8.7 || ^9.5 || ^10.4.14 || ^11.5",
-    "typo3/cms-extbase": "^8.7 || ^9.5 || ^10.4.14 || ^11.5",
-    "typo3/cms-backend": "^8.7 || ^9.5 || ^10.4.14 || ^11.5",
-    "typo3/cms-reports": "^8.7 || ^9.5 || ^10.4.14 || ^11.5",
-    "typo3/cms-frontend": "^8.7 || ^9.5 || ^10.4.14 || ^11.5"
+    "php": "^8.0 | ^8.2 | ^8.3",
```

Comment:
Irgendwie habe ich im Kopf dass das immer zwei Striche sein müssen, wie bei einem "Oder". Sieht komisch aus und ist in anderen Extensions auch anders

Reply:
Fixed, thx.